### PR TITLE
Consent versioning v0.02 nci

### PR DESCRIPTION
**Pull request by ramavavi**
Git hub issue: https://github.com/episphere/connectApp/issues/491

Changed the version1.0 for NCI HIPAA and Consent to Version0.02.

### CODE REVIEW

**Comment by anthonypetersen** [comment]
I will review this today, but please wait to merge until I give the green light.

**Comment by ramavavi** [comment]
Ok sure Tony, thank you.

File: js/pages/consent.js [hunk 1]
```diff
@@ -1073,6 +1073,7 @@ const consentSubmit = async e => {
     formData['558435199'] = 353358909;
     //consent and hipaa forms
     let versionJSON = await fetch('./forms/Consent_versioning.json').then(res => res.json());
+    console.log
```
Comment (anthonypetersen) [inline js/pages/consent.js:1076]: please remove

**Review by anthonypetersen — CHANGES_REQUESTED** [review]

**Comment by anthonypetersen** [comment]
Also need accompanying PDFs for these files (NCI versions at the very least)

**Comment by anthonypetersen** [comment]
is this still needed? if not, please close

**Pull request closed without merging**